.gitmodules

```diff
@@ -14,3 +14,7 @@
 	path = third_party/RejsekortReader
 	url = https://github.com/mchro/RejsekortReader.git
 	shallow = true
+[submodule "third_party/protobuf"]
```

Comment:
It doesn't look like this has a matching `third_party/protobuf` object here -- is this against `v3.9.1`?